projects/foreign_testing/orchestration/foreign_testing.py

```
@@ -0,0 +1,60 @@
+    repo_name = os.environ.get("REPO_NAME")
+    pull_pull_sha = os.environ.get("PULL_PULL_SHA")
+
+    if not foreign_repo_id:
+        raise ValueError("PSAP_FORGE_FOREIGN_TESTING must be set (and point to `foreign_testing.$NAME` field)")
+
+    foreign_repo_configs = config.project.get_config("foreign_testing")
+    foreign_repo_config = foreign_repo_configs.get(foreign_repo_id, None)
+    if foreign_repo_config is None:
+        raise ValueError(f"PSAP_FORGE_FOREIGN_TESTING must point to `foreign_testing.{foreign_repo_id}` field")
+
+    tmp_dest = f"/tmp/{repo_name}"
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)